release/rancher/rancher.go

```diff
@@ -10,7 +10,6 @@ import (
 	"fmt"
 	htmlTemplate "html/template"
```

Comment:
I don't think this needs an alias.

Reply:
In this file we use both `text/template` and `html/template`.